docs-new/docs/use-cases/non-k8s.md

```diff
@@ -107,7 +128,9 @@ kubectl get keptntasks -n my-keptn-annotated-namespace
 kubectl get pods -n my-keptn-annotated-namespace
 ```
 
-## Re-run the KeptnTask
+See the
```

Comment:
I think there is something missing, or this line can be removed

Reply:
I agree -- that is more confusing than helpful.  It's deleted